n2y/notion.py

```diff
@@ -466,7 +466,7 @@ def download_file(self, url, page, block_id):
 
     def save_file(self, content, page, extension, block_id):
         block_id_chars = strip_hyphens(block_id)
-        page_title = sanitize_filename(page.title.to_plain_text())
```

Comment:
Wait, I don't think this should be slugify here. I think in general we do want underscores.